### ISSUE TITLE
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null when scrolling down

### ISSUE THREAD

**Issue by LowLevelMahn** [dinusv/qml-gantt#6]
cloned git, version: sha: 49625b4, 7 Mar 2016
QtCreator 4.13.1+Qt 5.15.1 (freshly installed)

i get following error message if i start your example in QtCreator and scroll down

```
09:02:04: Starting D:\projects\test_gantt\build-qmlgantt-Desktop_Qt_5_15_0_MSVC2019_64bit-Debug\sample\debug\sample.exe ...
QML debugging is enabled. Only use this in a safe environment.
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
qrc:/main.qml:59: TypeError: Cannot read property 'width' of null
...
```

**Comment by dinusv** [comment]
Pushed a fix in 17ed4c11c3. Let me know if this works for you.

**Comment by LowLevelMahn** [comment]
worked!
